pkg/controller/controller_instance_test.go

```diff
@@ -892,8 +900,10 @@ func TestPollServiceInstanceInProgressDeprovisioningWithOperationNoFinalizer(t *
 		t.Fatalf("pollInstanceInternal failed but not with expected error, expected %q got %q", "still in progress", err)
 	}
 
+	// there should have been 1 action to update the instance status with the last operation
+	// description
 	actions := fakeCatalogClient.Actions()
-	assertNumberOfActions(t, actions, 0)
+	assertNumberOfActions(t, actions, 1)
```

Comment:
We do, see for example https://github.com/kubernetes-incubator/service-catalog/pull/957/files#diff-34dbc217e78a01dffcd4dfcd5d973d3cR392